### ISSUE TITLE
Broken links to risk assessment parameters

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-documentation#474]
## Where to find the issue
https://github.com/corona-warn-app/cwa-documentation/blob/master/cwa-risk-assessment.md
"How does the Corona-Warn-App identify an increased risk?"

and translated version of this document in German
https://github.com/corona-warn-app/cwa-documentation/blob/master/translations/cwa-risk-assessment.de.md

## Describe the issue
All links to specific Current Configuration parameters in
https://github.com/corona-warn-app/cwa-documentation/blob/master/cwa-risk-assessment.md are broken (404 file not found error). (Also in the [German translated document](https://github.com/corona-warn-app/cwa-documentation/blob/master/translations/cwa-risk-assessment.de.md).)

Here are the broken links from the end of the document:
- [Exposure Configuration](https://github.com/corona-warn-app/cwa-server/blob/master/services/distribution/src/main/resources/master-config/exposure-config.yaml)
- [Attenuation & Duration Configuration](https://github.com/corona-warn-app/cwa-server/blob/master/services/distribution/src/main/resources/master-config/attenuation-duration.yaml)
- [App Configuration, including minimum risk score](https://github.com/corona-warn-app/cwa-server/blob/master/services/distribution/src/main/resources/master-config/app-config.yaml)
- [Risk Score Classification](https://github.com/corona-warn-app/cwa-server/blob/master/services/distribution/src/main/resources/master-config/risk-score-classification.yaml)

## Suggested change
https://github.com/corona-warn-app/cwa-server/pull/1032 from Dec 3, 2020 by @sorin-stefan-iovita-sap has caused master-config to be renamed main-config, so the right links would now be:

- [Exposure Configuration](https://github.com/corona-warn-app/cwa-server/blob/master/services/distribution/src/main/resources/main-config/exposure-config.yaml)
- [Attenuation & Duration Configuration](https://github.com/corona-warn-app/cwa-server/blob/master/services/distribution/src/main/resources/main-config/attenuation-duration.yaml)
- [App Configuration, including minimum risk score](https://github.com/corona-warn-app/cwa-server/blob/master/services/distribution/src/main/resources/main-config/app-config.yaml)
- [Risk Score Classification](https://github.com/corona-warn-app/cwa-server/blob/master/services/distribution/src/main/resources/main-config/risk-score-classification.yaml)

**Comment by MikeMcC399** [comment]
It seems there was some parallel work going on!

https://github.com/corona-warn-app/cwa-documentation/pull/473 from @towo would fix this issue.

**Comment by towo** [comment]
Sorry, I just utterly skipped looking at issues and just checked if there's any open PRs related to the documentation changes. 😅

Amusing we stumbled on it roughly concurrently.

**Comment by MikeMcC399** [comment]
@towo 
> Sorry, I just utterly skipped looking at issues and just checked if there's any open PRs related to the documentation changes. 😅
> 
> Amusing we stumbled on it roughly concurrently.

No need to apologize! We were literally working in parallel and had no way of knowing about each other's work. I did check that there were no relevant issues and PRs before I started writing up my issue, but admittedly I didn't check again before I clicked on "Submit new issue"!

**Comment by MikeMcC399** [comment]
Fixed by https://github.com/corona-warn-app/cwa-documentation/pull/473
